docs/examples/opentelemetry-example-app/src/opentelemetry_example_app/grpc/route_guide_server.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# Copyright 2020, OpenTelemetry Authors
```

Comment:
Copyright should be updated to match https://github.com/open-telemetry/opentelemetry-python/pull/523